toolkit/docs/building/building.md

```diff
@@ -184,95 +206,125 @@ sudo make iso CONFIG_FILE=./imageconfigs/full.json REBUILD_TOOLS=y DOWNLOAD_SRPM
 sudo make image CONFIG_FILE=./imageconfigs/core-container.json REBUILD_TOOLS=y DOWNLOAD_SRPMS=y
 ```
 
-## ISOs
+### ISOs
+
 ISO installers can be built with:
+
 ```bash
 # Build out/images/developer_iso/*.iso from remote components
 sudo make iso -j$(nproc) CONFIG_FILE=./resources/imageconfigs/developer_iso/developer_iso.json
 ```
+
 To create an unattended ISO installer (no interactive UI) use `UNATTENDED_INSTALLER=y`:
-````bash
+
+```bash
 # Build out/images/developer_iso/*.iso from remote components with unattended installer
 sudo make iso -j$(nproc) CONFIG_FILE=./resources/imageconfigs/developer_iso/developer_iso.json UNATTENDED_INSTALLER=y
-````
+```
+
 NOTE: ISOs require additional packaging and build steps (such as the creation of a separate `initrd` installer image used to install the final image to disk).
 
-## Packages
+### Packages
+
 The toolkit can download packages from remote RPM repositories, or build them locally. By default any `*.spec` files found in `SPECS_DIR="./SPECS"` will be built locally. Dependencies will be downloaded as needed. Only those packages needed to build the current config will be built (`core-efi.json` by default). An additional space separated list of packages may be added using the `PACKAGE_BUILD_LIST=` variable.
 
 Build all local packages needed for the default `core-efi.json`:
+
 ```bash
 sudo make build-packages -j$(nproc)
 ```
 
 Build only two packages along with their prerequisites (note `CONFIG_FILE` is explicitly cleared, not specifying it will use the default `core-efi.json` config):
+
 ```bash
 sudo make build-packages PACKAGE_BUILD_LIST="vim nano" CONFIG_FILE= -j$(nproc)
 ```
 
 Build packages from a custom SPECS dir:
+
 ```bash
 sudo make build-packages SPECS_DIR="/my/packages/SPECS" -j$(nproc)
 ```
 
-### Working on Packages
+#### Working on Packages
+
 The build system will attempt to minimize rebuilds, but sometimes it is useful to force packages to rebuild, or ignore missing packages. Say you want to iterate on the `nano` package, but the `ncurses-devel` package is broken (`ncurses-devel` is a dependency of `nano`)...
 
-#### DOWNLOAD_SRPMS
+##### DOWNLOAD_SRPMS
+
 When `DOWNLOAD_SRPMS=y` is set, the local sources and spec files will not be used, and changes will not be reflected in the final packages.
 
-#### Force Rebuilds
+##### Force Rebuilds
+
 Adding `PACKAGE_REBUILD_LIST="nano"` will tell the build system to always rebuild `nano.spec` even if it thinks the rpm file is up to date.
 
-#### Source Hashes
+##### Ignoring Packages
```

Comment:
The `dev` branch no longer supports the `PACKAGE_IGNORE_LIST` so this section no longer applies. 

From: https://github.com/microsoft/CBL-Mariner/pull/141
>With the new tooling changes in this PR, support for PACKAGE_IGNORE_LIST was removed. This flag initially helped us deal with corner cases in spec files and the tooling. Between this tooling change PR and the accompanying spec files change PR (will be merged at the same time) there is no longer a need for this flag. Any problematic specs or tooling that resulted in packages incorrectly rebuilding have been fixed.
>
>**Going forward, any changes to CBL-Mariner that result in a package incorrectly rebuilding should be considered a build break and fixed before being checked in.**

Reply:
removed